lib/contentful_rich_text.dart

```diff
@@ -156,6 +156,11 @@ class ContentfulRichText {
     String? uri,
   }) {
     if (node['nodeType'] == 'hyperlink' || uri?.isNotEmpty == true) {
+      // Quickfix for as long as the link is missing
+      if (node['data']['uri'] == null) {
+        node['data']['uri'] = 'https://bahn.de';
+      }
+
```

Comment:
Maybe you can set the uri to `about:blank` instead of bahn.com as not all links would be related to DB. 
Can you also check if the `onTap` is set to default to a blank page in hyperlink.dart of contentful lib? that's the change i had made and wanted to know if this double setting is required or if they are two different errors.

Reply:
It's not enough to do it in `hyperlink.dart`. Without this fix it would crash at this point: `String link = uri ?? node['data']['uri'];`

But I think it's a good idea to set the value to `about:blank`.